src/configs/passport.ts

```diff
@@ -0,0 +1,66 @@
+import { Request } from 'express';
+import passport from 'passport';
+import passportLocal from 'passport-local';
+import passportJWT from 'passport-jwt';
+import { getRepository, Repository } from 'typeorm';
+import { User } from '../entities/user-entity';
+import { isCorrect } from '../services/user-crypto-service';
+
+const LocalStrategy = passportLocal.Strategy;
+const JWTStrategy = passportJWT.Strategy;
+const ExtractJWT = passportJWT.ExtractJwt;
+
+const cookieExtractor = (reg: Request): string => {
+  let token = '';
+  if (reg.cookies) {
+    token = reg.cookies.jwt;
+  }
+  return token;
```

Comment:
In my opinion it's redundant. Could you please rewrite something like this?
return reg.cookies ? reg.cookies.jwt : '';